cli: TestTenantZip failed

cli.TestTenantZip [failed](https://tanzanite.cluster.engflow.com/invocations/default/19dfd477-45eb-499e-9a9e-a83180fa097a?testReportRun=7&testReportShard=12&testReportAttempt=1#targets-Ly9wa2cvY2xpOmNsaV90ZXN0) on release-24.1.1-rc @ [0415ebc7ded1107eb98a98d93332e59b08f544c5](https://github.com/cockroachdb/cockroach/commits/0415ebc7ded1107eb98a98d93332e59b08f544c5):


```
=== RUN   TestTenantZip
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestTenantZip1544311698
    test_log_scope.go:81: use -show-logs to present logs inline
    zip_tenant_test.go:116: -- test log scope end --
test logs left over in: outputs.zip/logTestTenantZip1544311698
--- FAIL: TestTenantZip (133.41s)
=== RUN   TestTenantZip/testzip_shared_process_virtualization
[debug --host=127.251.190.169:35377 --insecure=true zip --concurrency=1 --cpu-profile-duration=1s /dev/null]
    datadriven.go:144: 
        /var/lib/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/cli/cli_test_/cli_test.runfiles/com_github_cockroachdb_cockroach/pkg/cli/testdata/zip/testzip_shared_process_virtualization:1:
         
        output didn't match expected:
        @@ -124,10 +124,11 @@
         [node ?] ? heap profiles found
         [node 1] requesting goroutine dump list... received response... done
         [node ?] ? goroutine dumps found
         [node 1] requesting cpu profile list... received response... done
         [node ?] ? cpu profiles found
        +[node 1] retrieving cpuprof.2024-06-04T09_29_04.770.97.pprof... received response... done
         [node 1] requesting log files list... received response... done
         [node ?] ? log files found
         [node 1] requesting ranges... received response... done
         [node 1] writing ranges... writing JSON output: debug/nodes/1/ranges.json... done
         [cluster] pprof summary script... writing binary output: debug/pprof-summary.sh... done
    --- FAIL: TestTenantZip/testzip_shared_process_virtualization (116.67s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=7</code>
 - <code>shard=12</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #124554 cli: TestTenantZip failed [C-test-failure O-robot P-2 T-observability branch-release-24.1 release-blocker]
</p>
</details>
/cc @cockroachdb/obs-prs @cockroachdb/server
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestTenantZip.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>


Jira issue: CRDB-39205

--- a/pkg/cli/zip_test.go
+++ b/pkg/cli/zip_test.go
@@ -620,6 +620,8 @@ func eraseNonDeterministicZipOutput(out string) string {
 	out = re.ReplaceAllString(out, `[node ?] ? goroutine dumps found`)
 	re = regexp.MustCompile(`(?m)^\[node \d+\] \d+ cpu profiles found$`)
 	out = re.ReplaceAllString(out, `[node ?] ? cpu profiles found`)
+	re = regexp.MustCompile(`(?m)^\[node \d+\] retrieving cpuprof.*$` + "\n")
+	out = re.ReplaceAllString(out, ``)
 	re = regexp.MustCompile(`(?m)^\[node \d+\] \d+ log files found$`)
 	out = re.ReplaceAllString(out, `[node ?] ? log files found`)
 	re = regexp.MustCompile(`(?m)^\[node \d+\] retrieving (memprof|memstats|memmonitoring).*$` + "\n")

